conda-forge.yml

```
@@ -1,5 +1,7 @@
 conda_forge_output_validation: true
-build_platform: {osx_arm64: osx_64}
-conda_forge_output_validation: true
-provider: {linux_aarch64: default, linux_ppc64le: default, win: azure}
+build_platform:
+  osx_arm64: osx_64
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```

what happened to `win`?

```suggestion
  linux_ppc64le: default
  win: azure
```

---

Windows now defaults to azure so this key shouldn’t be set anymore. 
It’s just a leftover thing.

---

Windows now defaults to azure so this key shouldn’t be set anymore. 
It’s just a leftover thing.